src/Components/DeleteMovieModal/DeleteMovieModal.scss

```diff
@@ -0,0 +1,3 @@
+.lol {
```

Comment:
Better to use to use a more meaningful name :)

Reply:
Sorry, it is a mistaken style which should not exist:)